examples/_debug/source/geojson/noCartoDbId.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>colorBins with common GeoJSON</title>
+
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css"
+    />
+
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://libs.cartocdn.com/airship-style/v2.4/airship.min.css"
+    />
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+  </head>
+
+  <body>
+    <section id="map"></section>
+
+    <!-- Map libraries-->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js"></script>
+    <script src="https://unpkg.com/deck.gl@8.2.0/dist.min.js"></script>
+    <script src="https://libs.cartocdn.com/airship-components/v2.4/airship.js"></script>
+
+    <script src="/dist/umd/index.min.js"></script>
+
+    <script>
+      async function initializePolygons() {
+        carto.auth.setDefaultCredentials({ username: 'public' });
+        const data = await fetch('./data.json').then(response => response.json());
+
+        const deckMap = carto.viz.createMap({
+          basemap: 'voyager',
+          view: {
+            longitude: 0,
+            latitude: 0,
+            zoom: 1
+          }
+        });
+
+        // style
+        const style = carto.viz.style.colorBins('value', {
+          size: 30,
+          palette: 'ORYEL',
+          viewport: true
+        });
+
+        // geojson source (external, to allow custom options)
+        const geojsonSource = new carto.viz.source.GeoJSON(data);
+        // const geojsonSource = new carto.viz.source.GeoJSON(data, { uniqueIdProperty: 'name' });
```

Comment:
For this example, should you uncomment line 66 and comment line 65?

Reply:
Well, it's a different use case... The main fix is the first one, using a geojson file without 'cartodb_id' (that was working wrong on interactivity). Adding a uniqueProperty to GeoJSON is indeed an option I'm now considering if it makes sense to expose or not.